robusta_krr/core/integrations/kubernetes/__init__.py

```
@@ -497,42 +612,82 @@ async def _list_all_groupedjobs(self) -> list[K8sObjectData]:
+                                name=job.metadata.name,
+                                namespace=job.metadata.namespace
+                            )
+                            # Store lightweight job info only for grouped jobs
+                            grouped_jobs[group_key].append(lightweight_job)
+                            # Keep only ONE full job as template per group
+                            if group_key not in grouped_jobs_template:
+                                grouped_jobs_template[group_key] = job
+                    if not continue_ref:
+                        break
+        
+        except Exception as e:
```

Ruff (0.14.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)